src/test/run-pass-fulldeps/compiler-calls.rs

```diff
@@ -12,6 +12,10 @@
 
 // ignore-cross-compile
 
+// FIXME: The proc-macro tests should work for stage1 when #49219 is merged.
+//        See also #49352.
```

Comment:
You should remove the FIXME - this uses the full compiler API, not `proc_macro`.